jython-processors/trigonometric-transformation/processor.json

```
@@ -45,15 +43,50 @@ The way to read the measure on the resulting trigonometric circle is then the sa
             "name": "periodical_column",
             "label": "periodiclal column",
             "type": "COLUMN",
```

The plugin should handle multiple columns 
```suggestion
            "type": "COLUMNS",
```

---

solved by #89233d4

---

Edit done : We can now handle multiple columns.

---

Edit : reversed back to "COLUMN" , after another discussion with you